package.json

```diff
@@ -38,10 +38,14 @@
   "homepage": "https://github.com/cozy/cozy-interapp-sandbox#readme",
   "devDependencies": {
     "babel-preset-cozy-app": "^1.2.1",
+    "cozy-scripts": "^1.1.2",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
     "eslint": "5.9.0",
     "eslint-config-cozy-app": "1.1.4",
+    "eslint-config-prettier": "^3.3.0",
```

Comment:
Shouldn't we be able to remove this line as `eslint-config-cozy-app` provides this dependency ? (I am not sure at all)

Reply:
No, unfortunately, yarn miss this dep :(